gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Mod properties
-mod_version=1.89.1
+mod_version=1.90.0
```

Comment:
Can you revert this for the time being? I don't tend to bump this until I actually release, as this forces me to write (and check) the changelog.

Reply:
Should I also revert the version number in whatsnew.txt and changelog.txt ? (to force recheck with :checkRelease ?)

Reply:
Oh, yes please!

Reply:
> I don't know if you've seen the changes in #478?
Nop. I haven't seen changes in #478. I just tried to use it but I did not succeed.

> As far as enchantments goes, I wonder if it's worth doing [what Plethora does](https://github.com/SquidDev-CC/plethora/blob/07ccc4d9b2652641c9290795996f2cebc306e320/src/main/java/org/squiddev/plethora/integration/vanilla/meta/VanillaMeta.java#L131), rather than just returning the raw NBT. This way we're not tied to how Mojang (currently) represents this information, and allows us to also include the enchantment's display name.
Yep.

I just thought of something: If we send the metadata acquisition to the main thread, it will necessarily cause a pause in the execution of the Lua program until the next tick ?. Isn't that likely to break programs? (Thx Google Translate)

Reply:
>  it will necessarily cause a pause in the execution of the Lua program until the next tick ?. Isn't that likely to break programs? (

Yep, hence the `detailed` parameter. We'd preserve the existing no-delay functionality when it's missing, and return detailed information from the main tick if it's `true`.

Reply:
This morning I thought of something: what if we added not one but two additional parameters ? 
`getItemDetail([slot, [detailed = false, [unsafeFastMode = false])`
detailed : get even more details (enchants, embedded items, ...)
unsafeFastMode : false = ask main thread to get advanced details, true = execute in lua thread (with a **_BIG RED WARNING_** in the doc)

Reply:
No.